build/js/live-editor.output_pjs.js

```diff
@@ -1602,10 +1614,38 @@ window.PJSOutput = Backbone.View.extend({
             // but PImage may be mutable, so that might not work.
             this.imageCache[file] = img;
         }.bind(this));
-    },
+        
+        _.each(sounds, function(file) {
+            var fileMatch = /["']([A-Za-z0-9_\/-]*?)["']/.exec(file);
 
+            if (!fileMatch) {
+                return loaded();
+            }
 
-    soundCache: {},
+            file = fileMatch[1];
+            
+            if (this.soundCache[file]) {
```

Comment:
I have a couple of ideas, but I'd like to think through the pros/cons of each before proceeding.